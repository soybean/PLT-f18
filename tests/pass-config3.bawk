BEGIN {}
 
LOOP {
  print($1);
}
 
END {}
 
CONFIG {
  RS = "\n";
  FS = "  ";
}
