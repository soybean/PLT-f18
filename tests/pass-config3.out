Hello
Line
This
Wow
Here

